stable/awsebscsiprovisioner/templates/roles.yaml

```diff
@@ -187,6 +187,9 @@ rules:
   - apiGroups: ["snapshot.storage.k8s.io"]
     resources: ["volumesnapshots/status"]
     verbs: ["update"]
+  - apiGroups: ["apiextensions.k8s.io"]
```

Comment:
It is still weird why this permission is not set in the aws-csi-driver repository. Should we raise it there and propose this change in the upstream repo?

Reply:
yes, we should raise it upstream